pkgs/by-name/sk/skia/package.nix

```diff
@@ -62,7 +63,15 @@ stdenv.mkDerivation (finalAttrs: {
     vulkan-memory-allocator
   ];
 
-  configurePhase = ''
+  configurePhase = let
+    cpu =
+         ({ "x86_64" = "x64";
+            "i686" = "x86";
+            "arm" = "arm";
+            "aarch64" = "arm64";
+          }.${stdenv.hostPlatform.parsed.cpu.name}
+        or (throw "no entry for cpu: ${stdenv.hostPlatform.parsed.cpu.name}"));
```

Comment:
I think we can remove the throw. There isn't much difference between "no entry for cpu x" and "attribute x missing". Or we can improve the message: "skia does not support architecture x"